lambda/magick.js

```diff
@@ -36,16 +36,22 @@ exports.default = async (event, gmOptions, env) => {
   const resizeImages = await resize(rules, imageVehicle, storageKey, uuid, imageName, tempOriginal, appPath);
   const { error: newErr, converted } = await format(resizeImages);
   const types = await converted;
-  const dimensions = sizeOf(tempOriginal);
+
+  // Example output:
+  // /tmp/screen_shot_2021_02_01_at_09.png PNG 642x664 642x664+0+0 8-bit sRGB 666819B 0.000u 0:00.000
+  const identifyStdout = childProcess.spawnSync(appPath, ['identify', tempOriginal], { encoding: 'utf-8' }).stdout;
+  const identifyData = identifyStdout.split(' ');
+  const [imageWidth, imageHeight] = identifyData[2].split('x');
+
   const response = {
     error: newErr,
     types,
     imageName,
     uri: `${storageKey}/${uuid}/`,
     dimensions: {
-      originalHeight: dimensions.height,
-      originalWidth: dimensions.width,
-      aspectRatio: parseFloat((dimensions.width / dimensions.height).toFixed(2))
+      originalHeight: parseInt(imageHeight, 10),
```

Comment:
I like this a lot, but we should put "something" here in case magick doesn't have the size, or if for some reason the `identifyStdout` doesn't match the expected output.

Reply:
Added a check for errors before parsing stdout 👍